docs/resources/ecosystem/officials.md

```diff
@@ -682,7 +682,7 @@ Key features of ZIO Logging:
 In order to use this library, we need to add the following line in our `build.sbt` file:
 
 ```scala
```

Comment:
when you're already at it, could u please also change line 678 from 
`It has a first citizen _Logger Context_ implemented on top`
to
`It has a first class citizen _Logger Context_ implemented on top`

Reply:
@domdorn Since this pr is already merged, maybe its good idea to create another pr for this?